jpa/hibernate5_3/src/main/java/org/jboss/as/jpa/hibernate5/HibernatePersistenceProviderAdaptor.java

```diff
@@ -54,6 +55,7 @@
 
     @Override
     public void injectJtaManager(JtaManager jtaManager) {
+        WildFlyCustomJtaPlatform.setTransactionSynchronizationRegistry(jtaManager.getSynchronizationRegistry());
```

Comment:
Why wasn't this one changed too? https://github.com/wildfly/wildfly/blob/b21c0f8f7ebcc855a903a26409b609379306d701/jpa/hibernate5/src/main/java/org/jboss/as/jpa/hibernate5/HibernatePersistenceProviderAdaptor.java#L56-L60

Reply:
We are changing how we integrate with Hibernate ORM 5.3.  I don't believe that ORM 5.1 would support this approach but I'm not against doing the same if possible for ORM 5.1.